### PR TITLE
Add bio.tools/EDAM section to tool_xml.rst

### PR DESCRIPTION
**Pull request by supernord**

### CODE REVIEW

**Comment by supernord** [comment]
@nsoranzo

**Comment by nsoranzo** [comment]
Thanks @supernord ! It looks we already had some text about this a bit below, in the "Tool cross-references" and following subsection. Can you maybe merge your suggestions with the text there?

**Comment by supernord** [comment]
Thanks for the quick review @nsoranzo 😄 
I'll merge the sections as soon as possible.

**Comment by nsoranzo** [comment]
Sorry for not checking/remembering that we already had a (slightly outdated) recommendation!

**Comment by supernord** [comment]
@nsoranzo - done 😄 
please let me know what you think

File: docs/best_practices/tool_xml.rst [hunk 1]
```diff
@@ -52,6 +52,7 @@ the version of the underlying tool, only the minor version number shall be
 increased if this is likely to bring the two version in sync in a reasonable
 time.
 
+
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
```

File: docs/best_practices/tool_xml.rst [hunk 2]
```diff
@@ -108,22 +109,62 @@ a "sentence" like structure.
...
+    <command detect_errors="exit_code"><![CDATA[
+
+Please reference only the tool which is wrapped, not the dependencies.
+
+If a bio.tools entry does not exist, you should add an entry as follows:
+
+-   Create an account on bio.tools.
+-   Add a new tool, completing as much metadata as possible / available.
+-   Ensure that you add at least one EDAM ``Topic`` and one EDAM ``Operation``
+to the entry (see below).
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
-   Ensure that you add at least one EDAM ``Topic`` and one EDAM ``Operation``
    to the entry (see below).
```

File: docs/best_practices/tool_xml.rst [hunk 3]
```diff
@@ -108,22 +109,62 @@ a "sentence" like structure.
...
+
+Tool annotations (EDAM)
+---------------------
 
-EDAM Topics and Operations
---------------------------
+The bio.tools registry makes use of the `EDAM ontology <http://edamontology.org/page>`__
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
The bio.tools registry makes use of the `EDAM`_ ontology
```

File: docs/best_practices/tool_xml.rst [hunk 4]
```diff
@@ -108,22 +109,62 @@ a "sentence" like structure.
...
+to annotate tools with ``Data``, ``Format``, ``Operation`` and ``Topic`` terms.
 
 `EDAM`_ terms are used to give a description of the tool's scientific domain and the
-functionalities it provides. Help picking EDAM terms can be provided by:
+functionalities it provides. `EDAM browser <https://edamontology.github.io/edam-browser/>`__
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
functionalities it provides. The `EDAM browser <https://edamontology.github.io/edam-browser/>`__
```

File: docs/best_practices/tool_xml.rst [hunk 5]
```diff
@@ -108,22 +109,62 @@ a "sentence" like structure.
...
+makes the process of picking EDAM terms much easier.
+
+Once a bio.tools identifier is added as an ``xref`` in the tool wrapper, the easiest way
+to improve the available EDAM annotations is to update the bio.tools entry metadata.
+
+If you wish to add EDAM annotations directly to a Galaxy tool wrapper, the two options are:
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
Alternatively, you can add EDAM annotations directly to a Galaxy tool by adding the
`edam_topics <https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-edam-topics>`__
and `edam_operations <https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-edam-operations>`__
elements to the tool XML file. This approach is not recommended though, as it risks duplicating the
tool annotation work in multiple places.
```

File: docs/best_practices/tool_xml.rst [hunk 6]
```diff
@@ -108,22 +109,62 @@ a "sentence" like structure.
...
 
-- browsers and visualizers, most of which are listed on the `EDAM`_ webpage.
-- descriptions of tools available in `bio.tools`_, many of which include the topics and operations relevant for the tool.
+- identify the EDAM annotations available in existing `bio.tools`_ entries for your tools: these
+will include ``Topics`` and ``Operations`` considered relevant for the tool, or
+- if the tool is not in bio.tools, you will need to add a new tool and
+annotate the entry with the relevant EDAM annotations.
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
```

File: docs/best_practices/tool_xml.rst [hunk 7]
```diff
@@ -108,22 +109,62 @@ a "sentence" like structure.
...
 
-When picking EDAM terms, avoid `root terms` such as `Topic` and `Operation`, and pick the most specific terms available. If you feel like some terms are missing to describe the tool, do not hesitate to [ask for new terms](https://edamontologydocs.readthedocs.io/en/latest/getting_involved.html#suggestions-requests).
+When picking EDAM terms, avoid `root terms` such as `Topic` and `Operation`, and pick the
+most specific terms available. If you feel like the terms required to describe the tool are
+missing, do not hesitate to [ask for new terms](https://edamontologydocs.readthedocs.io/en/latest/getting_involved.html#suggestions-requests).
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
missing, do not hesitate to `ask for new terms <https://edamontologydocs.readthedocs.io/en/latest/getting_involved.html#suggestions-requests>`__.
```

**Review by nsoranzo — COMMENTED** [review]
Thanks @supernord, I've added some suggestions!

**Comment by supernord** [comment]
Thanks @nsoranzo - changes committed 😄

File: docs/best_practices/tool_xml.rst [hunk 8]
```diff
@@ -108,22 +108,61 @@ a "sentence" like structure.
...
-- browsers and visualizers, most of which are listed on the `EDAM`_ webpage.
-- descriptions of tools available in `bio.tools`_, many of which include the topics and operations relevant for the tool.
+Alternatively, you can add EDAM annotations directly to a Galaxy tool by adding the
+`edam_topics <https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-edam-topics>`__
+and `edam_operations <https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-edam-operations>`__
+elements to the tool XML file. This approach is not recommended though, as it risks duplicating the
+tool annotation work in multiple places.
 
-When picking EDAM terms, avoid `root terms` such as `Topic` and `Operation`, and pick the most specific terms available. If you feel like some terms are missing to describe the tool, do not hesitate to [ask for new terms](https://edamontologydocs.readthedocs.io/en/latest/getting_involved.html#suggestions-requests).
+When picking EDAM terms, avoid `root terms` such as `Topic` and `Operation`, and pick the
```
Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst]:
```suggestion
When picking EDAM terms, avoid "root terms" such as ``Topic`` and ``Operation``, and pick the
```

**Review by nsoranzo — APPROVED** [review]
Thanks @supernord !

**Review by bgruening — APPROVED** [review]
Thanks @supernord

**Pull request merged**